web/viewer.css

```diff
@@ -1206,6 +1206,8 @@ html[dir='rtl'] .outlineItem > .outlineItems {
   text-decoration: none;
   display: inline-block;
   min-width: 95%;
```

Comment:
Note: I purposely left this line as a fallback, in the event that the browser has trouble parsing `calc`.